BlueDBOnDisk/src/main/java/org/bluedb/disk/collection/BlueCollectionOnDisk.java

```diff
@@ -221,6 +227,15 @@ protected void ensureCorrectKeyTypes(Collection<BlueKey> keys) throws BlueDbExce
 		}
 	}
 
+	protected static BlueDbVersion determineVersion(CollectionMetaData metaData) throws BlueDbException {
+		BlueDbVersion version = metaData.getVersion();

```

Comment:
If we need to change BlueDbVersion to a String how much work is that? I don't know that I want to serialize the enum constant and have to keep track of every version we ever create.

Reply:
Do we want to set legacy collections to current? Or do we want to know that they were sometime before current?